Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,30 @@
+using UnityEngine;
+
+public sealed class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private CharacterController characterController;
+    [SerializeField] private float movingSpeed;
+
+
+    private Vector2 _moveDirection;
+    private float _horizontal;
+    private float _vertical;
+
+    private void Update()
+    {
+        SetInput();
+        SetMoveDirection();
+    }
+
+    private void SetInput()

```

Comment:
Calling this SetInput suggests it is a setter. Setters usually have parameters, seeing as you are filling up already existing variables with input, in this case I think I'd call this GetInput.